common/views/components/ArchiveBreadcrumb/ArchiveBreadcrumb.js

```diff
@@ -0,0 +1,302 @@
+// @flow
+import { useState, useEffect } from 'react';
+import { workLink } from '../../../services/catalogue/routes';
+import NextLink from 'next/link';
+import { classNames } from '../../../utils/classnames';
+import fetch from 'isomorphic-unfetch';
+import styled from 'styled-components';
+import DropdownButton from '../DropdownButton/DropdownButton';
+import Icon from '../Icon/Icon';
+
+const ArchiveBreadcrumbNav = styled.nav`
+  * {
+    font-family: 'helvetica neue' !important;
+    font-weight: normal !important;
+    font-size: 14px !important;
+  }
+
+  ul {
+    list-style: none;
+    margin: 0 0 10px;
+    padding: 0;
+    display: inline-flex;
+    align-items: center;
+  }
+
+  li {
+    a {
+      text-decoration: none;
+      display: flex;
+      align-items: baseline;
+    }
+
+    &.is-last a {
+      background: #333;
+      color: white;
+      padding: 3px 8px;
+      border-radius: 3px;
+    }
+
+    .icon {
+      margin-right: 5px;
+      position: relative;
+      top: 2px;
+    }
+
+    position: relative;
+    cursor: pointer;
+    margin-right: 20px;
+
+    &:after {
+      position: absolute;
+      right: -15px;
+      top: 50%;
+      transform: translateY(-50%);
+      font-family: monospace;
+      color: #ccc;
+      content: '>';
+    }
+
+    &:last-child {
+      &:after {
+        display: none;
+      }
+    }
+
+    ul {
+      display: block;
+
+      li {
+        margin-bottom: 1px;
+
+        a {
+          padding: 3px 8px;
+          border-radius: 3px;
+
+          &:hover {
+            background: #333;
+            color: white;
+          }
+        }
+
+        &.is-active a {
+          background: #333;
+          color: white;
+        }
+
+        &:after {
+          display: none;
+        }
+
+        &:last-child {
+          &:after {
+            display: none;
+          }
+        }
+      }
+    }
+  }
+`;
+
+type Props = {|
+  work: { id: string },
+|};
+
+const ArchiveBreadcrumb = ({ work }: Props) => {
+  function findTree(path, collection) {
+    const pathParts = path.split('/'); // ['PPCRI', 'A', '1', '1']
+    const pathsToChildren = pathParts
+      .reduce((acc, curr, index) => {
+        if (index === 0) return [pathParts[0]];
+
+        return [...acc, `${acc[index - 1]}/${curr}`];
+      }, [])
+      .slice(1); // ['PPCRI/A', 'PPCRI/A/1', 'PPCRI/A/1/1']
+
+    return pathsToChildren.reduce(
+      (acc, curr) => {
+        const foundItem = acc[0].children.find(i => i.path.path === curr);
+
+        return [
+          {
+            work: foundItem.work,
+            path: foundItem.path,
+            children: foundItem.children,
+          },
+          ...acc,
+        ];
+      },
+      [
+        {
+          work: collection.work,
+          path: collection.path,
+          children: collection.children,
+        },
+      ]
+    );
+  }
+  function makeCrumbs(tree) {
```

Comment:
Definitely worth thinking about pulling this out into a `collection` lib as there are some other helpers in the other components, which might make it easier to see how we want to manipulate collections.

But also know we're in testing, so probably not worth fixing now.